_posts/2018-07-17-its-a-lot-more-snappy.md

```diff
@@ -0,0 +1,172 @@
+---
+layout: post
+title: "It's A Lot More Snappy"
+date: 2018-07-17
+categories: PWA SPA JavaScript
+author: Rick Foxcroft
+image: img/repeatable-serverless-applications/less_servers.png
+---
+
+**It's A Lot More Snappy**
+_‘A real world application of PWA technology’_
+
+Recently we undertook the journey of building a PWA(progressive web app) for a client here in Perth.  The app, is a maintenance app that allows maintenance technicians to raise issues with equipment in geologically  isolated areas. Due to this isolation, it was important that app could function in an offline mode and to do so, the app needed to store some data on the device it was running on.
+
+Whilst building the app, we encountered some performance issues. Over time the app would gradually become unresponsive to the point where, it would take a few seconds to register a button click.
+
+## What Was The Problem?
+
+* The app had to deal with a somewhat large dataset. One of these datasets was 205509 objects to be exact. Let’s refer to these objects as Pokemon.
```

Comment:
"with a somewhat large dataset" - suggestion to remove somewhat.

Reply:
`The app had to deal with a large dataset, of which one in particular was 205509 objects.`

Pokémon. 😝